0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -1,82 +1,91 @@
 
-A *connector* is a system that facilitates *payments* across different ledgers. Connectors generate revenue from Interledger payments while accepting some risk.
+1. Transfer value from itself to a receiver or
+2. Exchange value from one type to another.
 
-In the Interledger model, connectors are described as separate logical systems even though the same entity may operate a ledger and a connector.
+When transferring value, the sender will first request payment details from the receiver, including the receiver's ILP address. Once it has those details, it will start sending Interledger packets using this ILP address as the destination. A large payment may be split into many smaller pieces and reassembled by the receiver. For more information on the overall payment process, see the section [Interledger Protocol Suite](#interledger-protocol-suite).
 
-A connector receives a local transfer on one ledger in exchange for making another local transfer on a different ledger. A single interledger payment may include multiple connectors and may traverse any number of ledgers.
+A sender may also exchange assets by acting as both sender and receiver. In this case, it would send one asset and receive another, effectively exchanging one for the other.
 
-If the ledgers represent different assets, the connectors set the exchange rate between the transfers. Connectors may generate revenue from the difference in value between incoming and outgoing transfers. Senders may request quotes from multiple connectors to determine the best price before sending a payment.
+Senders typically choose a connector and send all their traffic through this connector. If they are at any point dissatisfied with their connector, they may switch to a different one. Sophisticated senders may operate a connector of their own and maintain relationships with multiple connectors to minimize costs and maximize availability.
 
-Connectors *peer* with one another to exchange information used to determine the best route for a payment.
+### Receivers
+
+A *receiver* is a system which accepts incoming Interledger packets and provides a cryptographic receipt for each packet which is returned to the sender along the same path. For more information on the cryptographic exchange, see [Payment Flow](#payment-flow).
 
 **Interledger ensures that senders' funds are safe throughout an multi-hop payment and cannot be stolen by faulty or malicious connectors (see [Interledger Security](#interledger-security)).**
 
 ### The Interledger
 
-The Interledger protocol suite may be used to transact across any ledgers and connectors, whether they are public or private. There is no single network that all parties must connect to to use these protocols.
+The "Interledger protocol suite" can be used among any peered group of connectors, be it are public or private. There is no single network that all parties must connect to to use these protocols.
```

Comment:
> can be used among any peered group of connectors

This makes it sound like all of the connectors must be connected with one another, rather than there being a chain of peering or customer relationships between them

Reply:
changed to 'network of peered connectors'